opa-manager-operator/files/rest.yaml

```diff
@@ -0,0 +1,486 @@
+---
```

Comment:
I don't see any references to this file. Can it be removed?

Reply:
yes, this is a reminent